code/style.css

```diff
@@ -1,4 +1,190 @@
+/* Variables for easy access color scheme */
+:root {
+  --bg-color: #899878;
+  --txt-color: #121113;
+  --title-bg-color: #e4e6c3;
+  --box-bg-color: #f7f7f2;
+  --nav-underline-color: #57d6a3;
+}
+
+/* Custom font for site-name */
+@font-face {
+  font-family: Seagram;
+  src: url(./Seagram.ttf);
+}
+
 html {
-  background: #0025ff;
-  color: #fff;
+  background: var(--bg-color);
+  color: var(--txt-color);
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+a,
+a:visited {
+  text-decoration: none;
+  color: var(--txt-color);
+}
+
+/* Content box properties */
+header,
+.big-news,
+.small-news-card,
+.footer {
+  background-color: var(--box-bg-color);
+  border-radius: 10px;
+  padding: 10px;
+  /* Polaroid(?) boxshadow */
+  -webkit-box-shadow: 0 4px 6px rgba(0, 0, 0, 0.5);
+  -moz-box-shadow: 0 4px 6px rgba(0, 0, 0, 0.5);
+  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.5);
```

Comment:
What's the difference between each box-shadow? All of them have the same values (?).